themes/edx.org/lms/templates/header.html

```diff
@@ -69,37 +69,36 @@ <h2 class="course-header">
 
     % if user.is_authenticated():
       % if not course or disable_courseware_header:
-        % if not nav_hidden or show_program_listing:
           <nav aria-label="Main" class="nav-main">
             <ul class="left list-inline nav-global authenticated">
               % if not nav_hidden:
```

Comment:
Why don't you just go ahead and move this line to above line 72? That's where the actual <nav> tag starts. What do you say?